docs/cli/edgedb_ui.rst

```diff
@@ -0,0 +1,47 @@
+.. _ref_cli_edgedb_ui:
+
+
+=========
+edgedb ui
+=========
+
+Start the EdgeDB UI in your default browser.
+
+.. cli:synopsis::
+
+    edgedb ui [<options>]
+
+
+Description
+===========
+
+``edgedb ui`` is a terminal command used to start the EdgeDB UI in your default
+browser. Alternatively, it can be used to print the UI URL with the
+``--print-url`` option.
+
+The EdgeDB UI is a tool that allows you to graphically manage and query your
+EdgeDB databases. It contains a REPL, a textual and graphical view of your
+database schemas, and a data explorer which allows for viewing your data as a
+table.
+
+.. note::
+
+    The UI is served by default by development instances. To enable the UI on a
+    production instance, use the ``--admin-ui`` option with ``edgedb-server``
+    or set the ``EDGEDB_SERVER_ADMIN_UI`` :ref:`environment variable
+    <ref_reference_envvar_admin_ui>` to ``enabled``.
+
+
+Options
+=======
+
+The ``ui`` command runs on the database it is connected to. For specifying the
+connection target see :ref:`connection options <ref_cli_edgedb_connopts>`.
+
+:cli:synopsis:`--print-url`
+    Print URL in console instead of opening in the browser. This is useful if
+    you prefer to open the EdgeDB UI in a browser other than your default
+    browser.
+
+:cli:synopsis:`--no-server-check`
+    Don't probe the UI endpoint of the server instance.
```

Comment:
> Can we make a comment why check is needed.

As I can remember, this [was designed](https://github.com/edgedb/edgedb-cli/pull/808#issuecomment-1208468767) to provide a friendly error if the UI is not enabled on the server.

> I think it could mess up HTTP/HTTPs protocol, right?

If I get your question correctly, looks like this was also [discussed](https://github.com/edgedb/edgedb-cli/pull/808#discussion_r942434692), with the take-away that we assume HTTP will redirect to HTTPS and that's safe for simple probing.

Reply:
Good call., @tailhook. I have pushed a commit with an explanation.